jvb/src/main/java/org/jitsi/videobridge/AbstractEndpoint.java

```diff
@@ -424,6 +487,37 @@ public void addReceiver(String receiverId, VideoConstraints newVideoConstraints)
         }
     }
 
+    /**
+     * Notifies this instance that a specified received wants to receive the specified video constraints from the media
+     * source with the given source name.
+     *
+     * The receiver can be either another endpoint, or a remote bridge.
+     *
+     * @param receiverId the id that specifies the receiver endpoint.
+     * @param sourceName the name of the media source for which the constraints are to be applied.
+     * @param newVideoConstraints the video constraints that the receiver wishes to receive.
+     */
+    public void addReceiverV2(String receiverId, String sourceName, VideoConstraints newVideoConstraints)
```

Comment:
Can you please annotate `sourceName` with `@NotNull` or `@Nullable` as appropriate? I think it's expected to be `@NotNull` but you use it as nullable in `RelayMessageTransport`

Reply:
I've added the annotation, but didn't get any error about using nullable. Is this an IDE warning or something?

Reply:
oh I was using `!!` which would throw